Source/ACE/Entity/MovementData.cs

```diff
@@ -250,6 +169,29 @@ public MovementData ConvertToClientAccepted(uint holdKey, CreatureSkill run)
             return md;
         }
 
+        public void SetMovementStateFlag()
+        {
```

Comment:
It was working the way I had it, but it is not working in accessor.   I am trying to debug now.